jenkins/opensearch/Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
             agent {
                 docker {
                     label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
-                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdk14-node10.24.1-cypress6.9.1-20211005'
+                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028'
```

Comment:
Can we remove the dates from the image name? I don't see any use of it. We can use tags instead. So even external contributor can directly pull latest instead of wondering which one to use to build the artifacts.

Reply:
I disagree. The CI runner contains many runners for different purposes, a latest tag is useless as it cannot point to multiple images.

Reply:
I see that `centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028` is itself is a tag and `opensearchstaging/ci-runner` is the repo. So no way of putting latest